perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -659,6 +659,7 @@ public ApplicationFormItem addFormItem(PerunSession user, ApplicationForm form,
 		// set new properties back to object & return
 		item.setOrdnum(ordnum);
 		item.setId(itemId);
+		perun.getAuditer().log(user, "Application form item ID=" + form.getId() + " voID=" + form.getVo().getId() + " groupID=" + form.getGroup().getId() + " has been added");
```

Comment:
Hi, please always check if group is not null, since it's not mandatory part of an application form (unlike VO, which is always there). You can use one-liner like:
```java
...." voID=" + form.getVo().getId() + (form.getGroup() != null) ? (" groupID=" + form.getGroup().getId()) : "" + " has been added");
```

Otherwise messages look OK. Thanks.